### ISSUE TITLE
TrackFind client: Improve selection of subsequent attribute values

### ISSUE THREAD

**Issue by sveinugu** [hyperbrowser/genomic-hyperbrowser#123]

**Comment by sveinugu** [comment]
Hide GSuite table and related selection boxes (incl execute button) when a second (or more) attribute query has not been fully selected

**Comment by radmilak** [comment]
@sveinugu Should there be a message displayed as well? I am not sure, since the message formatting from validateAndReturnErrors looks very error-like and this is not really an error situation, so not sure if it looks that good in the GUI..

**Comment by sveinugu** [comment]
Yes I think so. We often use the error text also for what is really help/info messages, which raises the issue of whether ProTo should really have an additional solution for that. For now, errors are all we got and we tend to use it for info purposes also. So go ahead!
